clang/lib/Driver/Driver.cpp

```diff
@@ -5767,6 +5767,13 @@ class OffloadingActionBuilder final {
                 FileTableTformJobAction::COL_CODE);
             WrapperInputs.push_back(ReplaceFilesAction);
           }
+          if (!AddWrapper) {
```

Comment:
```suggestion
          if (SkipWrapper) {
```
`if-condition` is easier to read without negation.